samples/pipeline/message-signing/Core_7/Shared/SignatureVerificationBehavior.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using System.Security.Cryptography;
+using System.Threading.Tasks;
+using NServiceBus.Logging;
+using NServiceBus.Pipeline;
+
+#region SignatureVerificationBehavior
+
+class SignatureVerificationBehavior :
+    Behavior<IIncomingPhysicalMessageContext>
+{
+    static readonly ILog log = LogManager.GetLogger<SignatureVerificationBehavior>();
+
+    public override Task Invoke(IIncomingPhysicalMessageContext context, Func<Task> next)
+    {
+        using (var hmac = HMAC.Create("hmacsha256"))
```

Comment:
Shouldn't the header from 
https://github.com/Particular/docs.particular.net/pull/4529/files#diff-1ba212019583e2df226e04f2981af784R21 be used here?